test/local/account_routes.js

```diff
@@ -856,6 +931,7 @@ test('/account/login', function (t) {
         t.deepEqual(mockMetricsContext.stash.args[2][1], 'account.keyfetch', 'third call was for account.keyfetch')
       }).then(function () {
         mockMailer.sendVerifyLoginEmail.reset()
+        mockDB.createSessionToken.reset()
```

Comment:
Ahha, it looks like here's the whole in testing. It looks like for `always on`, `sendNewDeviceLoginNotification` and `sendVerifyLoginEmail` are both checked, but not `sendVerifyCode`